pkg/controller/tuned/tuned_controller.go

```diff
@@ -99,6 +99,63 @@ type ReconcileTuned struct {
 	cfgv1client     *configv1client.ConfigV1Client
 }
 
+func (r *ReconcileTuned) isOperandUpgrade(tuned *tunedv1.Tuned) (bool, error) {
+	dsManifest, err := r.manifestFactory.TunedDaemonSet()
+	if err != nil {
+		return false, fmt.Errorf("Couldn't build tuned DaemonSet: %v", err)
+	}
+
+	daemonset := &appsv1.DaemonSet{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Namespace: dsManifest.Namespace, Name: dsManifest.Name}, daemonset)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			// Tuned DaemonSet doesn't exist yet; assume it wasn't manually deleted
+			return false, nil
+		} else {
+			return false, fmt.Errorf("Failed to get DaemonSet: %v", err)
+		}
+	}
+	// Tuned DaemonSet exists; compare the manifest and current version of the operand image
+	imageTuned := ntoconfig.NodeTunedImage()
+	return imageTuned != daemonset.Spec.Template.Spec.Containers[0].Image, nil
```

Comment:
Is the image the differentiating factor for upgrades?

Reply:
Good question.  I'd say it doesn't have to be in theory, but it will be in practice.  Happy to accept better ways of establishing when the operator is going through an upgrade!  The worst can happen is not removing the secrets if we miss an operator upgrade.